src/main/java/com/exasol/adapter/dynamodb/queryresultschema/QueryResultTableSchemaBuilder.java

```diff
@@ -46,7 +52,7 @@ public Void visit(final SqlStatementSelect select) throws AdapterException {
         }
 
         @Override
-        public Void visit(final SqlSelectList selectList) throws AdapterException {
+        public Void visit(final SqlSelectList selectList) {
```

Comment:
Why do you use Void class here?

Reply:
SqlNodeVisitor is a generic visitor where you specify the return type of the visit methods. As I don't want to return anything, I use `Void`. `void` can't be used generic.